src/index.ts

```diff
@@ -65,7 +65,10 @@ export default class Plugin extends AbstractBtpPlugin {
 
   private _port: number
   private _wsOpts: WebSocket.ServerOptions
-  private _httpServer: http.Server | https.Server
+  // There's an extra underscore because AbstractBtpPlugin already has a property
+  // named `_httpServer`.
+  /* tslint:disable-next-line:variable-name */
+  private __httpServer: http.Server | https.Server
```

Comment:
I would lean more to having a more verbose name like `_miniAccountsHttpServer` but it is up to you.

Otherwise LGTM